crypto/asn1/bio_ndef.c

```diff
@@ -184,7 +184,7 @@ static int ndef_suffix(BIO *b, unsigned char **pbuf, int *plen, void *parg)
         return 0;
 
     derlen = ASN1_item_ndef_i2d(ndef_aux->val, NULL, ndef_aux->it);
-    if ((p = OPENSSL_malloc(derlen)) == NULL) {
+    if (derlen < 0 || (p = OPENSSL_malloc(derlen)) == NULL) {
         ASN1err(ASN1_F_NDEF_SUFFIX, ERR_R_MALLOC_FAILURE);
```

Comment:
That error message does not really match up. Is the only reason the call fail is for malloc reasons?

Reply:
It's the most plausible reason but not the only one.
I'll address this in the morning.

I suspect that coverity (almost _covert_ again) has added some new rules about negative return values.

Reply:
Changed to not produce an error message for this new case.
The underlying code sometimes adds an error message already.